package.json

```diff
@@ -40,7 +43,25 @@
     "typescript": "5.3.3"
   },
   "devDependencies": {
+    "@types/mdast": "^4.0.4",
+    "chokidar": "^3.6.0",
+    "concurrently": "^8.2.2",
     "contentlayer2": "^0.4.6",
-    "prettier": "^3.2.4"
+    "husky": "^9.1.4",
+    "ignore": "^5.3.1",
+    "lint-staged": "^15.2.7",
+    "mdast": "^3.0.0",
+    "prettier": "^3.2.4",
+    "remark": "^15.0.1",
+    "remark-frontmatter": "^5.0.0",
+    "remark-parse": "^11.0.0",
+    "remark-stringify": "^11.0.0",
+    "strip-indent": "^4.0.0",
+    "unified": "^11.0.5",
+    "unist-util-visit": "^5.0.0",
+    "vfile": "^6.0.2"
+  },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx,md,css,md,mdx}": "yarn prettier:fix"
```

Comment:
we should also check `yml` and `yaml` files for linting since we use them within the recently added courses section.

can you add it here and in the prettier scripts above too? 
looks like I forgot to add it in there before :)

lets also throw `json` in there too while we are at it!